testing/resources/test_repo/eleven_module/res_partner_unlink.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def unlink(self):
+        if self.name == 'explode':
+            raise RuntimeError
```

Comment:
```suggestion
            raise RuntimeError()
```

Same in other exceptions